public/app/core/components/UserEdit/UserSessions.tsx

```diff
@@ -0,0 +1,166 @@
+        return false;
+      }
+      return true;
+    });
+
+    this.setState({ sessions });
+  }
+
+  async loadUserSessions() {
+    const { userId } = this.props;
+    let sessions: UserSession[] = [];
+
+    if (userId) {
+      sessions = await getBackendSrv().get('/api/admin/users/' + userId + '/auth-tokens');
+    } else {
+      sessions = await getBackendSrv().get('/api/user/auth-tokens');
+    }
+
+    sessions = this.sortSessions(sessions);
+
+    sessions = sessions.map((session: UserSession) => {
+      return {
+        ...session,
+        seenAt: dateTime(session.seenAt).fromNow(),
+        createdAt: dateTime(session.createdAt).format('MMMM DD, YYYY'),
+      };
+    });
+
+    this.setState({ sessions });
+  }
+
+  render() {
+    const { adminMode } = this.props;
+    const { sessions } = this.state;
+    return (
+      <>
+        <h3 className="page-sub-heading">Sessions</h3>
+        {sessions.length ? (
+          <div>
+            <div className="gf-form-group">
+              <table className="filter-table form-inline">
+                <thead>
+                  <tr>
+                    <th>Last seen</th>
+                    <th>Logged on</th>
+                    <th>IP address</th>
+                    <th>Browser &amp; OS</th>
+                    <th />
+                  </tr>
+                </thead>
+                <tbody>
+                  {sessions.map((session: UserSession, index) => {
+                    return (
+                      <tr key={index}>
+                        <td>{session.isActive ? 'Now' : session.seenAt}</td>
+                        <td>{session.createdAt}</td>
+                        <td>{session.clientIp}</td>
+                        <td>
+                          {session.browser} on {session.os} {session.osVersion}
+                        </td>
+                        <td>
+                          <Button
+                            variant="danger"
+                            size="sm"
+                            onClick={() => {
+                              this.revokeUserSession(session.id);
+                            }}
+                          >
+                            <i className="fa fa-power-off" />
+                          </Button>
+                        </td>
+                      </tr>
+                    );
+                  })}
+                </tbody>
+              </table>
+            </div>
+            {adminMode && (
+              <div className="gf-form-group">
+                <Button
+                  variant="danger"
+                  onClick={() => {
+                    this.revokeAllUserSessions();
+                  }}
+                >
+                  Logout user from all devices
+                </Button>
+              </div>
+            )}
+          </div>
+        ) : (
+          <p>No sessions found.</p>
+        )}
+      </>
+    );
+  }
+}
+
+export default UserSessions;
```

Comment:
don't use default exports. Named exports have some big downsides so we try to limit them as much as possible, only needed for redux components (ie top level page components)